src/tests/ftest/network/zero_config.py

```diff
@@ -45,7 +45,8 @@ def get_device_info(self):
         try:
             # Find any ib* device in the listing and initially use default numa and domain values
             for index, interface in enumerate(re.findall(r"ib\d", "\n".join(results[0]["stdout"]))):
-                self.interfaces[interface] = {"numa": index, "domain": "hfi1_{}".format(index)}
+                self.interfaces[interface] = {
+                    "numa": index, "domain": "hfi1_{}".format(index), "port": "1"}
```

Comment:
(pylint-consider-using-f-string) Formatting a regular string which could be a f-string